src/types.ts

```diff
@@ -20,18 +20,36 @@ export interface RTDBMetadata {
 }
 
 export interface Config {
-  project: 'testnet' | 'mainnet'
+  project: Environment
   databaseUrl: string
 }
 
-export interface RawTokenInfo {
-  address: string
-  decimals: number
+export interface TokenInfo {
+  address?: string
   name: string
   symbol: string
+  decimals: number
   imageUrl?: string
+  isNative?: boolean
   isCoreToken?: boolean
   pegTo?: string
+  isSupercharged?: boolean
+}
+
+export type TokenInfoDTO = TokenInfo & {
+  tokenId: string
```

Comment:
Do we absolutely need `tokenId`? Could we do without it by adding a `network` property instead?

Reply:
we'll have network if the cloud function's API maps network to arrays of token info. so no, we don't actually need tokenId.

Reply:
I realized we probably *should* add tokenId, for two reasons:
1. both blockchain-api and the wallet will use tokenId, and need to agree on which token is referred to by tokenId. They both query address-metadata, so I think it should be computed at the source to avoid errors in computing it in slightly different ways
2. Although the cloud function will likely group tokens by network, it will probably use the Network enum I defined, which is environment-agnostic. If you computed a token ID from this, there could be collisions between testnet and mainnet, kind of weird. I opted for adding environment-specific tokenId's using the NetworkId enum instead, per Joe's encouragement [here](https://github.com/valora-inc/address-metadata/pull/237#pullrequestreview-1613844915)